### PR TITLE
README improvements: probe-run, linker file

### PR DESCRIPTION
**Pull request by hn3000**
fix typos, add instructions for multiple probes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,30 @@ user@host ~/neotron-pico-bios $ DEFMT_LOG=debug cargo run --release
...
+[0]: Picoprobe CMSIS-DAP (VID: 2e8a, PID: 000c, Serial: Exxxxxxxxxxxxxx6, CmsisDap)
+[1]: STLink V2 (VID: 0483, PID: 3748, Serial: 0xxxxxxxxxxxxxxxxxxxxxxE, StLink)
+user@host ~/neotron-pico-bios $ PROBE_RUN_PROBE='0483:3748' DEFMT_LOG=debug cargo run --release
+```
+
+You can also just provide the probe Serial, for example if you have multiple 
+identical probes.
+
+The documentation for this feature can be found at 
+https://github.com/knurling-rs/probe-run#12-multiple-probes
```
Comment (thejpster) [inline README.md:109]:
```suggestion
<https://github.com/knurling-rs/probe-run#12-multiple-probes>
```

File: README.md [hunk 2]
```diff
@@ -86,6 +86,30 @@ user@host ~/neotron-pico-bios $ DEFMT_LOG=debug cargo run --release
...
+[1]: STLink V2 (VID: 0483, PID: 3748, Serial: 0xxxxxxxxxxxxxxxxxxxxxxE, StLink)
+user@host ~/neotron-pico-bios $ PROBE_RUN_PROBE='0483:3748' DEFMT_LOG=debug cargo run --release
+```
+
+You can also just provide the probe Serial, for example if you have multiple 
+identical probes.
+
+The documentation for this feature can be found at 
+https://github.com/knurling-rs/probe-run#12-multiple-probes 
+ --
```
Comment (thejpster) [inline README.md:110]: What does this line do?

File: README.md (hunk 1 above)
Comment (hn3000) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (hn3000) [reply README.md]: Umm, I remember wanting to add something, but after testing it didn't seem important.

**Pull request merged**